uk_election_timetables/election.py

```diff
@@ -99,3 +99,16 @@ def timetable(self) -> List[Dict]:
 
     def _calendar(self):
         return self.BANK_HOLIDAY_CALENDAR.from_country(self.country)
+
+    def get_date_for_event_type(self, event):
+        for e in self.timetable:
+            if e["event"] == event.name:
+                return e["date"]
+
+    def is_before(self, event):
+        today = datetime.now(timezone.utc).date()
+        return self.get_date_for_event_type(event) < today
```

Comment:
I also think we need to use `<=` on this one, but not the other one. At the code is currently written both will return `False` on the date of the timetable item